### ISSUE TITLE
Adding custom screen sizes seems to break display of current viewport

### ISSUE THREAD

**Issue by dsone** [jorenvanhee/tailwindcss-debug-screens#6]
Not sure if this is an actual bug or not but here it goes:
TailwindCSS version: 1.8.10, as well as 1.9.6
Used bundler: laravel-mix@5.0.9

When I add any custom screen size, the [index.js at line 2](https://github.com/jorenvanhee/tailwindcss-debug-screens/blob/master/index.js#L2) gets 
```
{
  sm: '640px',
  md: '768px',
  lg: '1024px',
  xl: '1280px',
  xs: '430px'
}
```
as input. "xs" is my custom size. I could name it whatever I want, put it wherever and however I want in my tailwind.config.js. Only my custom size, adding in default sizes as well, putting my custom size between the default ones.
No matter what, the custom screen size is always last place.
Meaning that [foreaching the sizes](https://github.com/jorenvanhee/tailwindcss-debug-screens/blob/master/index.js#L30) to create the classes will add the custom viewport last in the created CSS file, overwriting any previous style.
In turn, of course, I only see "screen: xs" in each viewport.

I am not sure why that behaviour occurs. It's probably because of the way TWCSS creates/merges the config but have no clue how TWCSS does it. Perhaps it's save to assume screen sizes are always in px? If so it could be circumvented by sorting the incoming screen sizes before creating the classes.

Anyway, for now I am monkey patching `.sort((a, b) => parseInt(a[1])-parseInt(b[1]))` after the `filter` in line 29 to get a correct sorting.

**Comment by jorenvanhee** [comment]
Hi! I guess you're using `theme.extend.screens` to add your `xs` size? If so, then this closed PR might help you: #5. Let me know if your situation is any different.

**Comment by dsone** [comment]
That is correct. The PR exactly describes my issue.
But as far as I can see, Tailwind - in my case - has no issue with it as described [here by you](https://github.com/jorenvanhee/tailwindcss-debug-screens/pull/5#issuecomment-723477753). Yet I am currently only using media queries with the custom screen size and no other utilities, in the way the [documetnation](https://tailwindcss.com/docs/breakpoints#custom-media-queries) describes it - via extend. This is a bit confusing to me.

But moving the custom screen size and adding in the default ones in my `tailwind.config.js` does solve the issue at hand no matter where I put the custom size within the key `screens`.
```
module.exports = {
    // ...
    theme: {
        screens: {
	    xs: '430px',
            // default ones from tailwindcss/stubs/defaultConfig.stub.js
	    sm: '640px',
	    md: '768px',
	    lg: '1024px',
	    xl: '1280px'
	},
	extend: {
	    colors: {
                // ...
            }
        }
    }
};
```
So I think this can be closed.
Thank you!
